synapse/storage/databases/main/roommember.py

```diff
@@ -848,19 +847,21 @@ async def get_joined_users_from_state(
 
         assert state_group is not None
         with Measure(self._clock, "get_joined_users_from_state"):
-            return await self._get_joined_users_from_context(
-                room_id, state_group, state, context=state_entry
-            )
+            return await self._get_joined_users_from_context(state_group, state)
 
-    @cached(num_args=2, iterable=True, max_entries=100000)
     async def _get_joined_users_from_context(
```

Comment:
given that there is no longer any `context`, this is an increasingly silly name.

I think it is now only called from `get_joined_users_from_state` - can we just inline it?

Reply:
Much better! https://github.com/matrix-org/synapse/pull/13569/commits/63967da1d790ceb3d4702b91614858b7950b0781